tap_snapchat_ads/client.py

```diff
@@ -162,6 +161,9 @@ def get_access_token(self):
         if response.status_code >= 500:
             raise Server5xxError()
 
+        if response.status_code == 429:
+            raise Server429Error()
+
```

Comment:
Why is this exception raised here, is the below "raise_for_error" function not raised this exception?

Reply:
No, 429 error code is not added in `ERROR_CODE_EXCEPTION_MAPPING`, thus it will raise `SnapchatError` rather than `Server429Error`.